### PR TITLE
adding new fields: date created, date released, file type, and pages

### PR DESCRIPTION
**Pull request by geramirez**
Updated model change for new designs.
@khandelwal  do we need a unique field to prevent documents from being loaded twice or should we use some type extraction from the `doc_file`.

### CODE REVIEW

**Comment by khandelwal** [comment]
I have some date stuff in another, earlier branch (that also had lots of other related stuff). I'm going to wait to merge this until I have a little bit of time to figure out what else is in that branch, and to resolve accordingly.

**Comment by geramirez** [comment]
@khandelwal cool, I'll be working on adding tests to the doc_procesing_toolkit in the meantime. 

Also, projectkrang/foia-hub/pull/11 should be ready to merge. We might have to merge from upstream/master a couple more times, but this is a start.

File: docusearch/models.py [hunk 1]
```diff
@@ -28,7 +28,14 @@ class Document(models.Model):
     text = models.TextField(
         null=False, help_text='The full text of the document')
     title = models.TextField(null=True)
-    date = models.DateField(null=True)
+    date_created = models.DateField(
```
Comment (khandelwal) [inline docusearch/models.py:37]: This is exactly how I made the date changes in an earlier branch. I'll fix mine to match yours I think. I called them 'create_date' and 'release_date' but that doesn't matter as much.

File: docusearch/models.py [hunk 2]
```diff
@@ -28,7 +28,14 @@ class Document(models.Model):
...
+        null=True, help_text='Date the document was created by agency')
+    date_released = models.DateField(
```
Comment (khandelwal) [inline docusearch/models.py:40]: Oh, we might not know the date the document was released, so I made this nullable. Eventually we'll know when we picked it up (and when we last looked at the website, data store) but making this nullable is going to give us options later.

**Comment by khandelwal** [comment]
This is a reasonable pull request. I just don't know if we'll always know the date a document was released - especially for documents that have already been released in the past.

**Comment by geramirez** [comment]
@khandelwal I though the release date was to show, when it was released on krang. Also, let me resolve the merge conflicts.

**Comment by khandelwal** [comment]
No because in the FOIA world the document release date means something else. That might get confusing otherwise.

**Comment by khandelwal** [comment]
@rjmajma Do you have any thoughts on release dates?

**Comment by rjmajma** [comment]
The release date would be to show when it was published online. I don't think it's helpful if we provide info on when we made it searchable/discoverable because it was already online before then.

**Comment by geramirez** [comment]
Ok, this should be ready now.

**Pull request merged**